### PR TITLE
Django 1.9

### PR DESCRIPTION
**Pull request by somexpert**

### CODE REVIEW

**Comment by somexpert** [comment]
ping @ldko, @madhulika95b, this is ready for review. Just brings the Django compatibility to 1.9

File: locking/admin.py [hunk 1]
```diff
@@ -53,21 +54,26 @@ def lock(self, obj):
         if obj.is_locked:
             seconds_remaining = obj.lock_seconds_remaining
             minutes_remaining = seconds_remaining/60
-            locked_until = _("Still locked for %s minutes by %s") \
```
Comment (ldko) [inline locking/admin.py:56]: If the `ugettext` calls are no longer needed, I don't think we need the import anymore.

**Comment by madhuBayy** [comment]
Other than what @ldko mentioned, everything else looks good.

File: locking/admin.py (hunk 1 above)
Comment (somexpert) [reply locking/admin.py:56]: We do not need those ugettext calls. I'll get rid of the import.

**Review by madhuBayy — APPROVED** [review]

**Review by ldko — APPROVED** [review]
:+1:

**Pull request merged**